src/main/java/org/littleshoot/proxy/impl/ClientToProxyConnection.java

```diff
@@ -177,7 +184,8 @@ public void operationComplete(
      **************************************************************************/
 
     @Override
-    protected ConnectionState readHTTPInitial(HttpRequest httpRequest) {
+    protected ConnectionState readHTTPInitial(ChannelHandlerContext ctx, Object httpRequestObj) {
```

Comment:
why did we change `HttpRequest httpRequest` to `Object httpRequestObj` ?

Reply:
yeah, I had some weird behavior with generic here..